rgw/v2/tests/s3_swift/configs/test_bucket_listing_fake_mp.yaml

```diff
@@ -0,0 +1,14 @@
+config:
```

Comment:
could you please update the polarion ID and script in the config

Reply:
done